app/controllers/users.ts

```diff
@@ -31,3 +24,37 @@ export function getUserById(req: Request, res: Response, next: NextFunction): Pr
     })
     .catch(next);
 }
+
+// eslint-disable-next-line consistent-return
+export async function createUser(req: Request, res: Response, next: NextFunction): Promise<Response | void> {
+  const { firstName, lastName, email, password } = req.body;
+
+  if (!firstName || !lastName || !email || !password) {
+    return res
+      .status(HttpStatus.BAD_REQUEST)
```

Comment:
here you should use next() function to pass the error to the errorHandler middleware. And you should throw a custom error, check the app/errors.js file